src/playbooks/with_build/upgrade_nuhs.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
For consistency's sake, let's capitalize the beginning of each of the task names here. We can also use "HA" instead of "hs"

Reply:
Fixed the capitalization. By "HA" to "hs", did you mean I should change the filename to upgrade_ha_nuh.yml?

Reply:
Done!